pkg/test/utils.go

```diff
@@ -102,6 +102,10 @@ func CompareGoldenFile(t *testing.T, p string, got string, normalizers ...func(s
 		if diff := cmp.Diff(want, got); diff != "" {
 			t.Errorf("unexpected diff in %s: %s", p, diff)
 		}
+
+		if err := os.WriteFile(p+"_2.log", []byte(got), 0644); err != nil {
```

Comment:
Did you intend for this to be here?  (I'm not following what it is for TBH)

Reply:
Nope! Good catch